src/renderer/components/shared/CanvasDataBuilder.js

```diff
@@ -1,6 +1,16 @@
 /* eslint-disable no-await-in-loop */
 import QuerySettings from '../Visualiser/RightBar/SettingsTab/QuerySettings';
 import { META_LABELS, baseTypes } from './SharedUtils';
+import store from '../../store';
+
+const convertToRemote = async (concept) => {
```

Comment:
I don't think this needs to be `async`, `asRemote` is not async and there are no `await`s.

Reply:
right! I realised that halfway through the changes lol
fixing this ...

Reply:
resolved in https://github.com/graknlabs/workbase/pull/275/commits/ab3a3c3bf2056939e5765f29692d9c066263f69a